tests/system/node/0/config.yaml

```diff
@@ -15,6 +15,10 @@ node:
   # Path to the data directory (if the path doesn't exist it will be created)
   data_dir: .cache
 
+  commons_budget_address: GCOMMONBGUXXP4RFCYGEF74JDJVPUW2GUENGTKKJECDNO6AGO32CUWGU
```

Comment:
This seems to be an unrelated change. Please move it to another commit, or remove it entirely.